code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/MessagingConstants.java

```diff
@@ -407,4 +407,20 @@ public class PayloadKeys {
             private PayloadKeys() {}
         }
     }
+
+    public final class AepUIKeys {
```

Comment:
Can you name these UIKeys to be consistent with the rest of the classes in this file? Also its specific to messaging, not sure why we need `Aep` prefix here